features/FxA-38-openid-connect/README.md

```diff
@@ -0,0 +1,36 @@
+# OpenID Connect
+
+https://mozilla.aha.io/features/FXA-38
+
+## Stories
+
+As a developer of a relying service, I want to delegate authentication and user profile data to Firefox Accounts via a standard protocol with good documentation and library support, so that I can focus on building my service without learning about a bespoke authentication protocol.
+
+## Details
+
+[OpenID Connect][spec] is a small layer on top of OAuth2. We are already largely compatible with OAuth2, and so the only addition needed to make us compliant with OpenID Connect is to add a `openid` scope, and return an `id_token` when that scope is requested.
+
+The `id_token` is a [JWT][jwt], with the following example:
+
+```json
+{
+  "iss": "https://oauth.accounts.firefox.com",
```

Comment:
I think `accounts.firefox.com` would be the best. (I know the auth server is at the `api` subdomain, but I prefer without it.)